registry/index.adoc

```diff
@@ -0,0 +1,31 @@
+[id="registry-overview"]
+= {product-title} registry overview
+include::modules/common-attributes.adoc[]
+:context: registry-overview
+
+toc::[]
+
+{product-title} can build images from your source code, deploy them, and manage
+their lifecycle. To enable this, {product-title} provides an internal,
+integrated container image registry that can be deployed in your {product-title}
+environment to locally manage images.
```

Comment:
Hmm, the overview looks like a statement for the internal registry, but not for all the kinds of registries. Please ingore it if I am wrong

Reply:
@bmcelvee - What's your take on this?

Reply:
Yes, that is correct! The original intention of the registry book was to focus on the internal registry because our other registry options (like Quay) are documented separately. However, we had a _Registry Options_ page that worked well with the overview content to address the registry options an OpenShift customer is likely to use. 

It might help to add a statement to clarify that this applies to the internal Image Registry, which is the focus of the Registry book, and that the _Registry Overview_ contains reference information and links for registries commonly used with OpenShift.

Reply:
Thank you for the clarification.  I will make the suggested change